code/game/objects/items/candle.dm

```diff
@@ -6,6 +6,7 @@
 	item_state = "candle1"
 	w_class = 1
 	heat_production = 1000
+	light_color = LIGHT_COLOR_TUNGSTEN

```

Comment:
Candles don't make tungsten light, they make flame light.
It's a different colour.